vault/hcp_link/proto/meta/meta.proto

```diff
@@ -34,6 +34,57 @@ message ListAuthResponse {
   repeated Auth Auths = 1;
 }
 
+message GetClusterStatusRequest {}
+
+message HANode {
+  string Hostname = 1;
+}
+
+message RaftServer {
+  // NodeID is the name of the server
+  string NodeID = 1;
+
+  // Address is the IP:port of the server, used for Raft communications
```

Comment:
I did not follow up with this after we spoke last time. Can we get an accurate info on the IP of the Raft node now?

Reply:
Not in the way that we discussed. At the node-level, that will be provided as `ListenerAddresses` and is provided in a separate [PR](https://github.com/hashicorp/vault/pull/18203). This will be the addresses as seen by the peers list.